.ci/kustomize/overlays/prod/elasticsearch-deployment.yaml

```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: elasticsearch
+spec:
+  template:
+    spec:
+      containers:
+      - name: elasticsearch
+        env:
+        - name: discovery.type
+          value: "single-node"
+        - name: ES_JAVA_OPTS
+          value: "-Dlog4j2.formatMsgNoLookups=true -Xms4g -Xmx4g"
```

Comment:
Good to see this in place. Is 4GB the right size for it? Seems excessive for how little it's used / it stores, but I know it can be resource hungry

Reply:
Reduced to 2G, as discussed